README.md

```diff
@@ -237,6 +237,38 @@ SDK_VERSION=2.6.5
 LOG_LEVEL=INFO
 ```
 
+## Extras
+
+The following optional services are also available:
+
+* Participant Query Store (PQS)
+
+To launch these additional services, follow these steps:
+
+1. Launch the other services, as described in the previous sections.
+2. Run the following script, which is required because the DARs must be uploaded _before_ starting the PQS.  
+   ```
+   scripts/upload-test-dars.sh
+   ```
```

Comment:
Is this step necessary. It appears step 3 uploads the dars prior to generating load?

Reply:
I agree it is not necessary. However, I felt it was helpful to explicitly include it since the DARs must currently be uploaded _before_ starting the PQS. Yes, the next step happens to _also_ upload the DARs.